store/index-refactor.js

```diff
@@ -2,11 +2,16 @@
 import Vue from "vue";
 
 export const state = () => ({
-    columnToCategoryMapping: {}
+    columnToCategoryMapping: {},
```

Comment:
We should alphabetize object keys if we don't have a functional grouping for them.

Reply:
Good catch @jarmoza. Didn't we have something setup to complain / fix this issue as part of the linting? Should we?